### DISCUSSION TITLE
How long to wait before posting an issue regarding an out-of-date app?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ScoopInstaller/Main#6285] [Q&A]
If an app's Scoop manifest points to an outdated version, and there is no existing issue report regarding that app, how long is it considered appropriate to wait until posting an issue report?

**Comment by Ra2-IFV** [comment]
CI runs every 4 hours, if the manifest has the `autoupdate` field. if not just open an issue

**Reply by Ra2-IFV** [reply]
but in my opinion you shouldn't open an issue for this if the autoupdate feature is not broken...
